GliaWidgets/SecureConversations/Welcome/SecureConversations.WelcomeStyle.swift

```diff
@@ -920,5 +1295,15 @@ extension SecureConversations.WelcomeStyle {
             /// Accessibility is not supported intentionally.
             public static let unsupported = Self(isFontScalingEnabled: false)
         }
+
+        mutating func apply(
+            configuration: RemoteConfiguration.Color?,
+            assetBuilder: RemoteConfiguration.AssetsBuilder
+        ) {
+            configuration?.value
+                .map { UIColor(hex: $0) }
+                .first
+                .unwrap { color = $0 }
+        }
```

Comment:
SwiftLint error because of line length

Reply:
fixed - 83157fb43a4418d4875c221c67224bb3f63cf139